src/EnergyPlus/HeatRecovery.cc

```diff
@@ -1439,11 +1467,58 @@ namespace HeatRecovery {
+					"Design Size Nominal Supply Air Flow Rate [m3/s]", NomSupAirVolFlowDes,
+					"User-Specified Nominal Supply Air Flow Rate [m3/s]", NomSupAirVolFlowUser );
+				if ( DisplayExtraWarnings ) {
+					if ( ( std::abs( NomSupAirVolFlowDes - NomSupAirVolFlowUser ) / NomSupAirVolFlowUser ) > AutoVsHardSizingThreshold ) {
+						ShowMessage( "Size:" + cHXTypes( ExchCond( ExchNum ).ExchTypeNum ) + ":Potential issue with equipment sizing for " +
+							ExchCond( ExchNum ).Name );
+						ShowContinueError( "User-Specified Nominal Supply Air Flow Rate of " + 
+							RoundSigDigits( NomSupAirVolFlowUser, 5 ) + " [m3/s]" );
+						ShowContinueError( "differs from Design Size Nominal Supply Air Flow Rate of " +
+							RoundSigDigits( NomSupAirVolFlowDes, 5 ) + " [m3/s]" );
+						ShowContinueError( "This may, or may not, indicate mismatched component sizes." );
+						ShowContinueError( "Verify that the value entered is intended and is consistent with other components." );
+					}
+				}
+			}
+		}
 
-			ReportSizingOutput( cHXTypes( ExchCond( ExchNum ).ExchTypeNum ), ExchCond( ExchNum ).Name, "Nominal Supply Air Flow Rate [m3/s]", ExchCond( ExchNum ).NomSupAirVolFlow );
+		if ( ExchCond( ExchNum ).ExchTypeNum == HX_AIRTOAIR_FLATPLATE ) {
+			IsAutoSize = false;
 
+			if ( ExchCond( ExchNum ).NomSecAirVolFlow == AutoSize ) {
+				IsAutoSize = true;
+			}
+			NomSecAirVolFlowDes = ExchCond( ExchNum ).NomSupAirVolFlow;
+ 
+			if ( IsAutoSize) {
+				ExchCond( ExchNum ).NomSecAirVolFlow = NomSecAirVolFlowDes;
+				ReportSizingOutput( cHXTypes( ExchCond( ExchNum ).ExchTypeNum ), ExchCond( ExchNum ).Name,
+					"Design Size Nominal Secondary Air Flow Rate [m3/s]", NomSecAirVolFlowDes );
+			} else {
+				if ( ExchCond( ExchNum ).NomSecAirVolFlow > 0.0 && NomSecAirVolFlowDes > 0.0 ) {
+					NomSecAirVolFlowUser = ExchCond( ExchNum ).NomSecAirVolFlow;
+					ReportSizingOutput( cHXTypes( ExchCond( ExchNum ).ExchTypeNum ), ExchCond( ExchNum ).Name,
+						"Design Size Nominal Secondary Air Flow Rate [m3/s]", NomSecAirVolFlowDes,
+						"User-Specified Nominal Secondary Air Flow Rate [m3/s]", NomSecAirVolFlowUser );
+					if ( DisplayExtraWarnings ) {
+						if ( ( std::abs( NomSecAirVolFlowDes - NomSecAirVolFlowUser ) / NomSecAirVolFlowUser ) > AutoVsHardSizingThreshold ) {
+							ShowMessage( "Size:" + cHXTypes( ExchCond( ExchNum ).ExchTypeNum ) + ":Potential issue with equipment sizing for " +
+								ExchCond( ExchNum ).Name );
+							ShowContinueError( "User-Specified Nominal Secondary Air Flow Rate of " + RoundSigDigits( NomSecAirVolFlowUser, 5 ) + " [m3/s]" );
+							ShowContinueError( "differs from Design Size Nominal Secondary Air Flow Rate of " + RoundSigDigits( NomSecAirVolFlowDes, 5 ) + " [m3/s]" );
+							ShowContinueError( "This may, or may not, indicate mismatched component sizes." );
+							ShowContinueError( "Verify that the value entered is intended and is consistent with other components." );
+						}
+					}
+				}
+			}
```

Comment:
I don't see anything that jumps out as being wrong. New sizing should be added using RequestSizing if possible since we do not want to propagate sizing code that needs to be maintained.